testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java

```diff
@@ -255,34 +243,16 @@ private static String executeProcess(final CliCommandBuilder commandBuilder, fin
         boolean wait = true;
         int runningTime = 0;
         int exitCode = Integer.MIN_VALUE;
-        do {
-            try {
-                Thread.sleep(STATUS_CHECK_INTERVAL);
-            } catch (InterruptedException e) {
-            }
-            runningTime += STATUS_CHECK_INTERVAL;
-            try {
+        try {
+            boolean finished = cliProc.waitFor(CLI_PROC_TIMEOUT, TimeUnit.MILLISECONDS);
```

Comment:
In the previous implementation, in case of timeout we had the trace LOGGER.info("A timeout has occurred while invoking CLI command."); If finished is false, you should resurrect this trace.